test/test_device_branch_constructors.jl

```diff
@@ -52,52 +44,228 @@ end
+
+    transformer = PSY.get_component(Transformer2W, system, "Trans4")
+    rate_limit2w = PSY.get_rate(tap_transformer)
+
+    for model in [DCPPowerModel, StandardPTDFModel]
+        template = get_template_dispatch_with_network(model)
+        op_problem_m = OperationsProblem(
+            template,
+            system;
+            optimizer = OSQP_optimizer,
+            PTDF = PSY.PTDF(system),
+        )
+        @test build!(op_problem_m; output_dir = mktempdir(cleanup = true)) ==
+              PSI.BuildStatus.BUILT
+
+        if model == DCPPowerModel
+            @test check_variable_bounded(op_problem_m, :Fp__HVDCLine)
+            @test check_variable_unbounded(op_problem_m, :Fp__TapTransformer)
+            @test check_variable_unbounded(op_problem_m, :Fp__Transformer2W)
+        end
+
+        psi_constraint_test(op_problem_m, ratelimit_constraint_names)
+
+        @test solve!(op_problem_m) == RunStatus.SUCCESSFUL
+
+        @test check_flow_variable_values(
+            op_problem_m,
+            :Fp__HVDCLine,
+            "DCLine3",
+            limits_min,
+            limits_max,
+        )
+        @test check_flow_variable_values(
+            op_problem_m,
+            :Fp__TapTransformer,
+            "Trans3",
+            -rate_limit,
+            rate_limit,
+        )
+        @test check_flow_variable_values(
+            op_problem_m,
+            :Fp__Transformer2W,
+            "Trans4",
+            -rate_limit2w,
+            rate_limit2w,
+        )
+    end
+end
+
+@testset "DC Power Flow Models Monitored Line Flow Constraints and Static Unbounded" begin
+    system = PSB.build_system(PSITestSystems, "c_sys14_dc")
+    hvdc_line = PSY.get_component(HVDCLine, system, "DCLine3")
+    limits_from = PSY.get_active_power_limits_from(hvdc_line)
+    limits_to = PSY.get_active_power_limits_to(hvdc_line)
+    limits_min = min(limits_from.min, limits_to.min)
+    limits_max = min(limits_from.max, limits_to.max)
+
+    tap_transformer = PSY.get_component(TapTransformer, system, "Trans3")
+    rate_limit = PSY.get_rate(tap_transformer)
+
+    transformer = PSY.get_component(Transformer2W, system, "Trans4")
+    rate_limit2w = PSY.get_rate(tap_transformer)
+
+    for model in [DCPPowerModel, StandardPTDFModel]
+        template = get_template_dispatch_with_network(model)
+        set_device_model!(template, DeviceModel(HVDCLine, HVDCUnbounded))
+        set_device_model!(template, DeviceModel(TapTransformer, StaticBranchBounds))
+        set_device_model!(template, DeviceModel(Transformer2W, StaticBranchBounds))
+        op_problem_m = OperationsProblem(
+            template,
+            system;
+            optimizer = OSQP_optimizer,
+            PTDF = PSY.PTDF(system),
+        )
+        @test build!(op_problem_m; output_dir = mktempdir(cleanup = true)) ==
+              PSI.BuildStatus.BUILT
+
+        if model == DCPPowerModel
+            # TODO: Currently Broken, remove variable bounds in HVDCUnbounded
```

Comment:
Is this broken too?

Reply:
yup, this is where PowerModels is adding a bounds in the unbounded formulation